packages/website/src/docs/configuration/tracing.md

```diff
@@ -0,0 +1,66 @@
+---
+id: tracing
+title: Tracing
+---
+
+# Tracing
+
+By default the Cadl Compiler will build without any debug information. The standard output will be minimal and limited to any `warning` or `error` diagnostics emitted during compilation.
+
+Some additional information is however being collected and can be revealed using the `--trace` cli flag.
+
+```bash
+cadl compile . --trace import-resolution
+```
+
+You can use the `--trace` option multiple times if there is multiple areas that should be logged from.
+
+```bash
+cadl compile . --trace import-resolution  --trace projection
+```
+
+## Trace selection
+
+The tracing system in the cadl compiler works by having each trace under an area. The area name is a dot `.` seperate string of area segments.
+
+When filtering which area to select you can use this area path to select which area is going to be revealed.
+The filter follow the same naming style, expect the last segment could be a wildcard `*`. This is however the same result as omitting the last segement all toghether. In other words, those filter have the exact same behavior:
```

Comment:
This makes me wonder why we have the `.*` syntax. Is it only at the top-level filter x != filter x.*?

Reply:
I guess it is really not needed, was just thinking that;s something that feel quite intuitive to be able to do